### PR TITLE
EIP-2: Grammar fixes, Meta ref and parameters

### PR DESCRIPTION
**Pull request by jamesray1**

### CODE REVIEW

File: EIPS/eip-2.md [hunk 1]
```diff
@@ -7,19 +7,30 @@ Type: Standard Track
...
 
-1. The gas cost *for creating contracts via a transaction* is increased from 21000 to 53000, ie. if you send a transaction and the to address is the empty string, the initial gas subtracted is 53000 plus the gas cost of the tx data, rather than 21000 as is currently the case. Contract creation from a contract using the `CREATE` opcode is unaffected.
-2. All transaction signatures whose s-value is greater than `secp256k1n/2` are now considered invalid. The ECDSA recover precompiled contract remains unchanged and will keep accepting high s-values - this is useful if e.g. a contract recovers old Bitcoin signatures.
-3. If contract creation does not have enough gas to pay for the final gas fee for adding the contract code to the state, the contract creation fails (ie. goes out-of-gas) rather than leaving an empty contract.
+1. The gas cost *for creating contracts via a transaction* is increased from 21,000 to 53,000, i.e. if you send a transaction and the to address is the empty string, the initial gas subtracted is 53,000 plus the gas cost of the tx data, rather than 2,1000 as is currently the case. Contract creation from a contract using the `CREATE` opcode is unaffected.
```
Comment (pirapira) [inline EIPS/eip-2.md]: `2,1000` is weird.

File: EIPS/eip-2.md [hunk 2]
```diff
@@ -7,19 +7,30 @@ Type: Standard Track
...
+2. All transaction signatures whose s-value is greater than `secp256k1n/2` are now considered invalid. The ECDSA recover precompiled contract remains unchanged and will keep accepting high s-values—this is useful e.g. if a contract recovers old Bitcoin signatures.
```
Comment (pirapira) [inline EIPS/eip-2.md]: I prefer `;` over `—`.

File: EIPS/eip-2.md (hunk 1 above)
Comment (jamesray1) [reply EIPS/eip-2.md]: Sorry that's a mistake.

File: EIPS/eip-2.md (hunk 2 above)
Comment (jamesray1) [reply EIPS/eip-2.md]: I think either is OK to use here.

**Pull request merged**